### ISSUE TITLE
Arm-based Docker image fails to boot.

### ISSUE THREAD

**Issue by azuchi** [chaintope/tapyrus-core#185]
When using the `linux/arm64`  image, the following error occurs:

```
tapyrusd: error while loading shared libraries: libboost_system.so.1.65.1: cannot open shared object file: No such file or directory
```

**Comment by Naviabheeman** [comment]
this happened because `arm64` tapyrusd container also used `x86-64` tapyrus-builder. #187 fixes the docker files to generate different containers.

New tapyrus/builder:[test_aarch64](https://hub.docker.com/layers/builder/tapyrus/builder/test_aarch64/images/sha256-c895058cbe17a448da8dab34feecbc336546492cd48d6706cacf0c80bb18d28e?context=explore)
New tapyrus/tapyrusd:[v0.5.1rc1](https://hub.docker.com/layers/tapyrusd/tapyrus/tapyrusd/v0.5.1rc1/images/sha256-06ed6fdd2ccd2053403467829e99fb6a2615baf4bf5e4bbba5e344cb130fbd39?context=explore)

`[ec2-user@ip-172-31-16-239 ~]$ sudo docker run -a STDOUT --name 'tapyrus_aarch64_node_testnet' -v tapyrus:/etc/tapyrus/ -e GENESIS_BLOCK_WITH_SIG='01000000000000000000000000000000000000000000000000000000000000000000000044cc181bd0e95c5b999a13d1fc0d193fa8223af97511ad2098217555a841b3518f18ec2536f0bb9d6d4834fcc712e9563840fe9f089db9e8fe890bffb82165849f52ba5e01210366262690cbdf648132ce0c088962c6361112582364ede120f3780ab73438fc4b402b1ed9996920f57a425f6f9797557c0e73d0c9fbafdebcaa796b136e0946ffa98d928f8130b6a572f83da39530b13784eeb7007465b673aa95091619e7ee208501010000000100000000000000000000000000000000000000000000000000000000000000000000000000ffffffff0100f2052a010000002776a92231415132437447336a686f37385372457a4b6533766636647863456b4a74356e7a4188ac00000000'  tapyrus/tapyrusd:v0.5.1rc1`
2022-04-08T19:42:28Z Tapyrus Core version v0.5.0.0-unk (release build)
2022-04-08T19:42:28Z InitParameterInteraction: parameter interaction: -whitelistforcerelay=1 -> setting -whitelistrelay=1
2022-04-08T19:42:28Z Validating signatures for all blocks.
2022-04-08T19:42:28Z Using the 'standard' SHA256 implementation
2022-04-08T19:42:28Z Default data directory /root/.tapyrus
2022-04-08T19:42:28Z Using data directory /var/lib/tapyrus/dev-1905960821
2022-04-08T19:42:28Z Using config file /etc/tapyrus/tapyrus.conf
2022-04-08T19:42:28Z Using at most 125 automatic connections (32768 file descriptors available)
2022-04-08T19:42:28Z Reading Genesis Block from [/var/lib/tapyrus/genesis.1905960821]
2022-04-08T19:42:28Z Genesis Block [038b114875c2f78f5a2fd7d8549a905f38ea5faee6e29a3d79e547151d6bdd8a] of Tapyrus network [1905960821] Loaded successfully
2022-04-08T19:42:28Z Using 16 MiB out of 32/2 requested for signature cache, able to store 524288 elements

**Comment by Naviabheeman** [comment]
fixed in v0.5.1
